src/UI/home/dashboard/home.tsx

```diff
@@ -1,34 +1,34 @@
-import React, {Component} from 'react'
-import {UserService} from '../../../backend-service-connector/service'
-import {UserMenu} from "../navigation/userMenu";
-import {HomeNavigation} from "../navigation/homeNavigation";
-import {HomeContext} from "./homeContext";
+import React, { Component } from 'react'
+import { UserMenu } from '../navigation/userMenu'
+import { HomeNavigation } from '../navigation/homeNavigation'
+import { HomeContext } from './homeContext'
+import { WalletService } from '../../../backend-service-connector/service'
 
 export class Home extends Component<any, any> {
-    userService: UserService
+    walletService: WalletService
 
     constructor(props: any) {
         super(props)
 
-        this.userService = props.userService
+        this.walletService = props.walletService
     }
 
     render() {
         return (
-            <HomeMenu userService={this.userService}/>
+            <HomeMenu walletService={this.walletService}/>
         )
     }
 }
```

Comment:
class-based component. refactor it to the functional component. [here's an article on the differences between those two](https://djoech.medium.com/functional-vs-class-components-in-react-231e3fbd7108). class components are legacy and now only functional components are used

Reply:
now I agree that functional components are awesome :D